src/tools/language-server/lspLoader.ts

```diff
@@ -10,14 +10,14 @@
 import fs from 'fs';
 import path from 'path';
 
-import {LanguageServiceContext} from './util.js';
+import {Logger} from './util.js';
 import {Loader} from '../../runtime/loader.js';
 
 export class LspLoader extends Loader {
```

Comment:
The file name seems not to follow our file naming conventions (`lsp-loader.ts`), is this intentional?

Reply:
Thanks, just an old habit, fix on its way.